habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -343,57 +385,72 @@ def act(
                     raise ValueError(
```

Comment:
I think we should update hl_info here, as we did in the SIRo branch

Reply:
Good point, I included this.